share/wake/lib/core/list.wake

```diff
@@ -811,7 +822,8 @@ export def unzip (list: List (Pair a b)): Pair (List a) (List b) = match list
 export def groupBy (cmp: a => a => Boolean) (list: List a): List (List a) =
     def sorted = sortBy cmp list
     def combo elem acc = match acc
-        (head, rest), tail if ! cmp elem head =
-            (elem, head, rest), tail
+        (head, rest), tail if ! cmp elem head = (elem, head, rest), tail
         _ = (elem, Nil), acc
     foldr combo Nil sorted
+
+
```

Comment:
Where did these newlines come from?

Reply:
mentioned in the meeting, its just a bug that hasn't been fixed yet